### PR TITLE
Dialogue Added

### PR DESCRIPTION
**Pull request by SabrinaTheCookie**
- Dialogue appears via
- - OnFerryingEnter
- - OnReturningEnter
- Easily modular to add in other events.
- Dialogue is currently linear rather than random
- - e.g. 1-2-3-4-5

- Added TempUI with temporary controls.
- Updated Prefabs

### CODE REVIEW

**Comment by TomasMorton** [comment]
Dialogue appears via
- OnFerryingEnter
- OnReturningEnter

and also when resuming after pause lol

File: Assets/Prefabs/UI/TempUI.prefab [hunk 1]
```diff
@@ -0,0 +1,272 @@
+%YAML 1.1
```
Comment (TomasMorton) [inline Assets/Prefabs/UI/TempUI.prefab:1]: What does a TempUI do?

Comment (SabrinaTheCookie) [reply Assets/Prefabs/UI/TempUI.prefab:1]: Just a textbox with the current temporary controls. This will be replaced later on.

**Comment by TomasMorton** [comment]
> Dialogue appears via
> 
> * OnFerryingEnter
> * OnReturningEnter
> 
> and also when resuming after pause lol

This is because transitioning from Pause you re-enter Returning or Ferrying

**Comment by SabrinaTheCookie** [comment]
> and also when resuming after pause lol

Fixed in latest commit

File: Assets/Scripts/Managers/DialogueManager.cs [hunk 2]
```diff
@@ -44,6 +44,9 @@ private void OnDisable()
 
     public void StartDialogue()
     {
+        //Dont run dialogue if the game was just paused.
+        if(GameStateManager.Instance.PreviousState == GameStateManager.GameStates.Pause) return;
```
Comment (TomasMorton) [inline Assets/Scripts/Managers/DialogueManager.cs:48]: That works 😂

**Review by TomasMorton — APPROVED** [review]

**Pull request merged**